libcxx/test/std/ranges/range.adaptors/range.enumerate/iterator/base.pass.cpp

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// <ranges>
+
+// class enumerate_view
+
+// class enumerate_view::iterator
+
+// constexpr const iterator_t<Base>& base() const & noexcept;
+// constexpr iterator_t<Base> base() &&;
+
+#include <array>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+#include <tuple>
+
+#include "test_iterators.h"
+#include "../types.h"
+
+template <class Iterator>
+constexpr void testBase() {
+  using Sentinel          = sentinel_wrapper<Iterator>;
+  using View              = MinimalView<Iterator, Sentinel>;
+  using EnumerateView     = std::ranges::enumerate_view<View>;
+  using EnumerateIterator = std::ranges::iterator_t<EnumerateView>;
+
+  auto make_enumerate_view = [](auto begin, auto end) {
+    View view{Iterator(begin), Sentinel(Iterator(end))};
```

Comment:
```suggestion
    View view{Iterator(std::to_address(begin)), Sentinel(Iterator(std::to_address(end)))};
```

Ditto for other constructions of `Iterator`.

Reply:
@frederick-vs-ja I applied your suggestion and the tests pass now. What puzzles me (and bothers me) is that no other tests in "ranges" do this type of expression. I don't know how to reproduce the error locally so I haven't experimented with it and I just took the fix in good faith.

Reply:
Oh, I found that existing test use helper functions that take containers and call `data()` and `size()`, like the following  (don't need to be well-constrained):
```C++
constexpr auto make_minimal_view(std::ranges::contiguous_range auto&& r) {
  return View{Iterator(std::ranges::data(r)), Sentinel(Iterator(std::ranges::data(r) + std::ranges::size(r)))};
}
```

But I think the `to_address` approach is equivalently OK, despite being not so conventional...